File: engine/engine-model/src/main/scala/net/server/channel/packet/mts/ChangePagePacket.scala
package net.server.channel.packet.mts

class ChangePagePacket(private var _available: Boolean, private var _operation: Byte, private var _tab: Int, private var _type: Int, private var _page: Int) extends BaseMTSPacket(_available = _available, _operation = _operation) {
  def tab: Int = _tab

  def theType: Int = _type

  def page: Int = _page
}

File: engine/engine-model/src/main/java/net/server/channel/packet/mts/ChangePagePacket.java
package net.server.channel.packet.mts;

public class ChangePagePacket extends BaseMTSPacket {
   private final Integer tab;

   private final Integer theType;

   private final Integer page;

   public ChangePagePacket(Boolean available, Byte operation, Integer tab, Integer theType, Integer page) {
      super(available, operation);
      this.tab = tab;
      this.theType = theType;
      this.page = page;
   }

   public Integer tab() {
      return tab;
   }

   public Integer theType() {
      return theType;
   }

   public Integer page() {
      return page;
   }
}
